Source/CarthageKit/Project.swift

```diff
@@ -458,6 +459,21 @@ public final class Project {
 				return copyProduct(dSYMURL, resolvedDestinationURL)
 			}
 	}
+	
+	/// Copies any *.bcsymbolmap files matching the given framework and contained
+	/// within the given directory URL to the directory that the framework
+	/// resides within.
+	///
+	/// If no bcsymbolmap files are found for the given framework, completes with
+	/// no values.
+	///
+	/// Sends the URLs of the bcsymbolmap files after copying.
+	public func copyBCSymbolMapsToBuildFolderForFramework(frameworkURL: NSURL, fromDirectoryURL directoryURL: NSURL) -> SignalProducer<NSURL, CarthageError> {
+		return BCSymbolMapsForFramework(frameworkURL)
```

Comment:
I think the implementation does not work for some archives those are not created from `carthage archive`, when a framework and its bcsymbolmap files are not in the same directory.